speeduino/corrections.ino

```diff
@@ -181,11 +181,21 @@ Additional fuel % to be added when the engine is cranking
 uint16_t correctionCranking()
 {
   uint16_t crankingValue = 100;
-  //if ( BIT_CHECK(currentStatus.engine, BIT_ENGINE_CRANK) ) { crankingValue = 100 + configPage2.crankingPct; }
+  //Check if we are actually cranking
   if ( BIT_CHECK(currentStatus.engine, BIT_ENGINE_CRANK) )
   {
     crankingValue = table2D_getValue(&crankingEnrichTable, currentStatus.coolant + CALIBRATION_TEMPERATURE_OFFSET);
     crankingValue = (uint16_t) crankingValue * 5; //multiplied by 5 to get range from 0% to 1275%
+
+  //If we're not cranking, check if if cranking enrichment tapering to ASE should be done
+  } else if ( (uint16_t) (currentStatus.runSecs*10) < configPage10.crankingEnrichTaper)
```

Comment:
You could use `else if ( runSecsX10) < configPage10.crankingEnrichTaper )` as runSecsX10 is used above, it will speed up the line.

Reply:
As discussed, runSecsX10 isn't an option here as it will overflow after 109 minutes.

Reply:
Just checked here, the runSecsX10 will run until 65535 and wont overflow as the code is: 
`if ( (BIT_CHECK(currentStatus.engine, BIT_ENGINE_RUN) ) && (runSecsX10 < 65535) ) { runSecsX10++; }`

Or I'm missing something and did that code wrong?

Reply:
The code is fine
```
if ( (BIT_CHECK(currentStatus.engine, BIT_ENGINE_RUN) ) && (runSecsX10 < 65535) ) { runSecsX10++; }
    else { runSecsX10 = 0; }
```
But at 65535 the if condition isn't true, so the else will put it into zero.